docs/developer/eventing/sources/kafka-source/README.md

```diff
@@ -324,62 +313,66 @@ You can specify the key deserializer among four types:
-        kind: Service
-        name: event-display
-   ```
+To specify the key deserializer, add the label `kafkasources.sources.knative.dev/key-type` to the
+`KafkaSource` definition, as shown in the following example:
+
+```yaml
+apiVersion: sources.knative.dev/v1beta1
+kind: KafkaSource
+metadata:
+name: kafka-source
+labels:
+  kafkasources.sources.knative.dev/key-type: int
+spec:
+consumerGroup: knative-group
+bootstrapServers:
+- my-cluster-kafka-bootstrap.kafka:9092 # note the kafka namespace
+topics:
+- knative-demo-topic
+sink:
+  ref:
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    name: event-display
+```
 
 ## Optional: Specify the initial offset
 
-By default the `KafkaSource` starts consuming from the `latest` offset in each partition. In case you want to consume from the earliest offset, set the initialOffset field to `earliest`.
-
-   ```yaml
-   apiVersion: sources.knative.dev/v1beta1
-   kind: KafkaSource
-   metadata:
-    name: kafka-source
-   spec:
-    consumerGroup: knative-group
-    initialOffset: earliest
-    bootstrapServers:
-    - my-cluster-kafka-bootstrap.kafka:9092 # note the kafka namespace
-    topics:
-    - knative-demo-topic
-    sink:
-      ref:
-        apiVersion: serving.knative.dev/v1
-        kind: Service
-        name: event-display
-   ```
+By default the `KafkaSource` starts consuming from the latest offset in each partition. If you want
+to consume from the earliest offset, set the initialOffset field to `earliest`, for example:
+
+```yaml
+apiVersion: sources.knative.dev/v1beta1
+kind: KafkaSource
+metadata:
+name: kafka-source
+spec:
+consumerGroup: knative-group
+initialOffset: earliest
+bootstrapServers:
+- my-cluster-kafka-bootstrap.kafka:9092 # note the kafka namespace
+topics:
+- knative-demo-topic
+sink:
+  ref:
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    name: event-display
+```
 
 !!! note
-    Valid values for `initialOffset` is `earliest` or `latest`, any other value would result in a validation error. This field will be honored only if there are no prior committed offsets for that consumer group.
+    The valid values for `initialOffset` are `earliest` and `latest`. Any other value results in a
+    validation error. This field is honored only if there are no prior committed offsets for that
+    consumer group.
 
-## Connecting to a TLS-enabled Kafka broker
+## Connecting to a TLS-enabled Kafka Broker
 
 The KafkaSource supports TLS and SASL authentication methods. To enable TLS authentication, you must have the following files:
 
 * CA Certificate
 * Client Certificate and Key
 
-KafkaSource expects these files to be in pem format, if it is in other format like jks, please convert to pem.
+KafkaSource expects these files to be in pem format, if it is in other format like jks, convert to pem.
 
 1. Create the certificate files as secrets in the namespace where KafkaSource is going to be set up, by running the commands:
 
```

Comment:
I'm wondering whether you need to run the lines after the `kubectl create secret` commands below, or whether they are output? If they are output, consider either deleting or putting them in a separate box because when you copy the command from the box, you end up copying both lines.

Reply:
Can you answer this, @matzew?

Reply:
Or @lberk (now that you're in [Knative Party](https://knative.party/)?)

Reply:
`secret/cacert created` and `secret/key create` are indeed output.

Reply:
Thanks, @lberk, I've deleted those lines to prevent confusion.